presto-hive-hadoop2/bin/common.sh

```diff
@@ -1,7 +1,3 @@
-#!/bin/bash
```

Comment:
This helps tools like vim to set coloring automatically (otherwise they don't know what shell flavor it is).
`#!bash` would be enough for vim specifically.

Reply:
Makes sense, I'll leave it in